assets/js/components/Work/Tabs/About/CoreMetadata.jsx

```diff
@@ -104,9 +115,20 @@ const WorkTabsAboutCoreMetadata = ({
       </div>
       <div className="column is-half">
         {/* License */}
-        <UIFormField label="License" mocked notLive>
+        <UIFormField label="License">
           {isEditing ? (
-            <p>Form elements go here</p>
+            <UIFormSelect
+              register={register}
+              name="license"
+              label="License"
+              options={licenseData ? licenseData.codeList : []}
+              defaultValue={
+                descriptiveMetadata.licenseStatement
+                  ? descriptiveMetadata.licenseStatement.id
+                  : ""
+              }
+              errors={errors}
+            />
```

Comment:
I think in standup you mentioned a "helper" default option for `select` form elements.  This is the `prop` which will give an empty initial value:

https://github.com/nulib/meadow/blob/81fee1555612b51228116a5c6ea69d866f830322/assets/js/components/UI/Form/Select.jsx#L14